.github/actions/analytics-uploader-wrapper/action.yaml

```diff
@@ -42,6 +45,7 @@ runs:
         token: ${{ inputs.token-prod }}
         cli-version: ${{ inputs.cli-version }}
         junit-paths: ${{ inputs.junit-paths }}
+        bazel-bep-path: ${{ inputs.bazel-bep-path }}
```

Comment:
Ordinarily we would remove line 47 as well, but because this is a reusable action in this repo, we can just add both and one will be passed in as an empty string